edweiss-app/app/(app)/(tabs)/notifications.tsx

```diff
@@ -96,85 +111,74 @@ const NotificationsTab: ApplicationRoute = () => {
     notifsYear.sort((a, b) => b.data.date.seconds - a.data.date.seconds);
     otherNotifs.sort((a, b) => b.data.date.seconds - a.data.date.seconds);
 
-
-
-
     return (
-        <>
-            <RouteHeader isBold title={"Notifications"}
-                right=
-                <TTouchableOpacity onPress={() => console.log("press parameters icon")}>
-                    <Icon name='cog' size={iconSizes.lg} testID={testIDs.parameters} mr={24} />
-                </TTouchableOpacity>
+        <TView>
+            <RouteHeader isBold title={t(`notifications:notifications`)}
+                right={
+                    <TTouchableOpacity testID={testIDs.parametersTouchable} onPress={() => console.log("press parameters icon")}>
+                        <Icon testID={testIDs.parameters} name='cog' size={iconSizes.lg} mr={24} />
+                    </TTouchableOpacity>
+                }
             />
-            <TView bb={0.5} mt={10} borderColor='crust'></TView>
+            <TView testID={testIDs.divider} bb={0.5} mt={10} borderColor='crust'></TView>
 
-            <TScrollView p={16} backgroundColor="transparent" testID={testIDs.scrollView}>
+            <TScrollView testID={testIDs.scrollView} p={16} backgroundColor="transparent">
                 {notifs.length > 0 ? (
-                    <TView mb={30}>
+                    <TView testID={testIDs.notifsView} mb={30}>
                         {notifsDay.length > 0 && (
-                            <>
-                                <TText mb={10} mt={15} ml={2} size={16} color='darkBlue' bold={true}>{t(`notifications:today`)}</TText>
+                            <TView testID={testIDs.todayView}>
+                                <TText testID={testIDs.todayText} mb={10} mt={15} ml={2} size={16} color='darkBlue' bold={true}>{t(`notifications:today`)}</TText>
                                 {notifsDay.map((notif, index) => (
-                                    <TView key={index} testID={testIDs.notifView}>
-                                        <NotifDisplay item={notif.data} id={notif.id} dateSection='today' index={notifs.indexOf(notif)} />
-                                    </TView>
+                                    <NotifDisplay item={notif.data} id={notif.id} dateSection='today' index={index} key={notif.data.title} />
```

Comment:
There is just a warning about the `key` prop. When you .map on each item and then `Display` it then React-native will take into account the `key` of each displayed item. So if the `key`s between 2 different items are identical, react-native will show a warning. Even though it does not affect the usability or display-ability of the code. Just to be safe, we must keep in mind to have a clean, without-warning product at the end.